src/chttpd.erl

```diff
@@ -1074,3 +1075,53 @@ respond_(#httpd{mochi_req = MochiReq}, Code, Headers, _Args, start_response) ->
     MochiReq:start_response({Code, Headers});
 respond_(#httpd{mochi_req = MochiReq}, Code, Headers, Args, Type) ->
     MochiReq:Type({Code, Headers, Args}).
+
+get_user(#httpd{user_ctx = #user_ctx{name = User}}) -> User;
+get_user(#httpd{user_ctx = undefined}) -> "undefined".
```

Comment:
Nvm, I see it. Thought it's strange that this value is unstable depending on what operation you proceed.